README.md

```diff
@@ -48,9 +48,11 @@ For development:
 Thanks
 ------
 
-PuppetModule.info is maintained by Dominic Cleal, and is unaffiliated to Puppet.
+PuppetModule.info was maintained by Dominic Cleal, and is unaffiliated to Puppet.
 RubyDoc.info was created by Loren Segal (YARD) and Nick Plante (rdoc.info) and is a project of DOCMETA, LLC.
 Additional help was provided by [YARD's friendly developer community](https://github.com/lsegal/rubydoc.info/graphs/contributors).
+
+Dominic ran the site for years, until he migrated it to Vox Pupuli.
```

Comment:
migrated or donated?

Reply:
I'm not sure what's the actual difference. I updated it to donated. let me know what you think about it.